UI/LinkText.h

```diff
@@ -182,6 +182,22 @@ std::string LinkTaggedIDText(std::string_view tag, int id, std::string_view text
 std::string LinkTaggedPresetText(std::string_view tag, std::string_view stringtable_entry,
                                  std::string_view display_text);
 
+/**
+ * Helper for generating a link string for a raw string with unknown tag
+ * by searching the various content managers. Returns unlinkified raw
+ * string if no match can be found.
+ */
+std::string LinkStringIfPossible(std::string raw, std::string user_string);
```

Comment:
this should probably take `const std::string&` or `string_view` as `LinkTaggedPresetText` just takes `string_view` so this will make unnecessary copies

Reply:
i'm still quite new to c++ and i tried to comply with the suggestions. should be less string copying now. also i removed dislikes-references to another disabled content.